### ISSUE TITLE
Assembly DLL name is reserved for internal use

### ISSUE THREAD

**Issue by col000r** [Nethereum/Unity3dSimpleSample#7]
Getting this error in Unity 2018.4.16f1: Assembly DLL name is reserved for internal use: Assets/Plugins/UnityEngine.dll (did files generated by a build accidentally end up in your Assets/ folder?)

**Comment by juanfranblanco** [comment]
Are you using the latest with the sample? Remove the UnityEngine.dll it is output as part of the build.

**Comment by col000r** [comment]
I just downloaded the zip and opened it in 2018.4.16f1. 
If I delete UnityEngine.dll, then I get `Assets/EtherTransfer.cs(32,27): error CS1729: The type 'Nethereum.JsonRpc.UnityClient.EthTransferUnityRequest' does not contain a constructor that takes '2' arguments`

**Comment by juanfranblanco** [comment]
Yes the sample needs updating to the latest version, but nothing major, you just need supply your chainId (if wanted) like mainnet = 1 or a null value https://github.com/Nethereum/Nethereum/blob/master/src/Nethereum.Unity/EthTransferUnityRequest.cs#L11
